library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java

```diff
@@ -57,10 +58,23 @@
 
     public HashMap<String, HashMap<String, String>> libraryModification = null;
 
+    public Class ownLibsActivityClass = null;
```

Comment:
Just set this to the `LibsActivity` by default. So you can remove the nullcheck on the bottom.